instagram/models/post.py

```
@@ -0,0 +1,72 @@
+
+    history = HistoricalRecords()
+
+    def __str__(self):
+        return f"{self.user.username} - {self.id}"
+
+    def generate_blur_data_url_task(self):
+        """
+        Generates a blurred data URL from the thumbnail_url using a Celery task.
+        This method queues the blur data URL generation as a background task.
+        """
+        from instagram.tasks import post_generate_blur_data_url  # noqa: PLC0415
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)